includes/class-sensei-db-query-learners.php

```diff
@@ -55,17 +55,23 @@ private function build_query() {
 				`u`.`user_login`,
 				`u`.`user_email`,
 				GROUP_CONCAT(`c`.`comment_post_ID`, '|', IF(`c`.`comment_approved` = 'complete', 'c', 'p' )) AS 'course_statuses',
-				COUNT(`c`.`comment_approved`) AS 'course_count'
+				COUNT(`c`.`comment_approved`) AS 'course_count', (
+					SELECT MAX(cm.comment_date_gmt)
+					FROM {$wpdb->comments} cm
+					WHERE cm.user_id = u.ID
+					AND cm.comment_approved IN ('complete', 'passed', 'graded')
```

Comment:
Should this also include "ungraded"? Does the Students report include that status?

Reply:
Great! Thanks a lot,  I had a little confusion regarding the completion state, now it's cleared :)

Reply:
But it should match the Students report, yes? 🙂